autopts/bot/iut_config/zephyr.py

```diff
@@ -121,6 +121,15 @@
         ]
     },
 
+    "multiple_iso_adv_sets.conf": {
+        "overlay": {
+            'CONFIG_BT_CTLR_ADV_ISO_SET': '2',
+	},
+	"test_cases": [
+	    'BAP/BSRC/SCC/BV-38-C',
+	]
+    },
```

Comment:
Is the plan to keep this as a long-term solution, or temporary? If long term, then I suggest that we set all the Kconfig options for the test here, and not just this one), so keep the hci_ipc_cpunet.conf file as small as possible for the the other tests. 

If a temporary solution until the bug in the ZLL is fixed, we can keep this as is

Reply:
Hard to say. So far I've only encountered this one test that requires two adv sets, but there might be others either in the future or simply not in the ICS. I would say the best solution would be to fix the bug in ZLL and put it back in the hci_ipc_cpunet.conf, so that all tests are using the same controller config

Reply:
Will let @sjanc determine how to best approach this :)

Reply:
Just realized that this needs to be set in the net-core and not app-core. This would set it in the app-core, right? @sjanc

Reply:
yeap, this affects only bttester app build (ie not netcore)
(overlays are generally meant for tuning IUT behavior to match test requirements, not to workaround bugs in IUT)

I think the right solution would be to create issue for this in zephyr and point that in errata/zephyr.yml so that it is listed test results